static/style.css

```diff
@@ -175,11 +175,37 @@ a:hover {
     font-weight: 900;
 }
 
+
 .TextFormattingH5{
     color: #d4388e;
     font-weight: 600;
 }
 
 .Width {
-    width: 500px;
+    width: 600px;
+}
+
+.MarginLeftRight {
+    margin-left: 10px;
+    margin-right: 10px;
```

Comment:
all the spacings from style guide are multiples of 8.
how about of update 10px for 8px?